.github/workflows/translate.yml

```
@@ -0,0 +1,53 @@
+name: Translate PyPy
+
+on:
+  push:
+    branches: [ "main", "py3.9", "py3.10"]
+  pull_request:
+    branches: [ "main", "py3.9", "py3.10"]
```

This [will add](https://docs.github.com/en/actions/using-workflows/manually-running-a-workflow) the ability to kick of this action via a button on the workflow page.

```suggestion
    branches: [ "main", "py3.9", "py3.10"]
  workflow_dispatch:    
```